### DISCUSSION TITLE
Qt and VS meanings

### DISCUSSION THREAD

**Discussion by ghost** [hasherezade/pe-bear#27] [General]
what's the difference between vs 17, vs 13, vs 10(qt4) versions? are the newer ones not compatible with older systems? if so, when does such compatibility starts/ends?

**Comment by hasherezade** [comment]
hi @cyberhardt !
Indeed it is related with backward compatibility. The Qt4 version runs on on Windows XP (some people still requested me to support this system, so I added). The VS13 version is self-contained - you don't have to install any redistributables package to be able to run it, all the libraries are shipped in one package. The VS17 version is just newer, but requires redistributable package to be installed on the system. Detailed description about what each version stands for were added in the past to all the releases, i.e. here: https://github.com/hasherezade/pe-bear-releases/releases/tag/0.5.5.7
At some point I stopped adding this note, because I thought it is a redundant information. But maybe I should attach it again.

**Reply by ghost** [reply]
no need to attach them on the release page, you can just put the info on the main page, it's easier to see and you won't need to repeat yourself for every release
